spec/grape/api_spec.rb

```diff
@@ -1133,7 +1133,7 @@ class DummyFormatClass
 
       subject.use Rack::Chunked
       subject.get('/stream') { stream test_stream }
-      get '/stream', {}, 'HTTP_VERSION' => 'HTTP/1.1'
+      get '/stream', {}, 'HTTP_VERSION' => 'HTTP/1.1', 'SERVER_PROTOCOL' => 'HTTP/1.1'
```

Comment:
@dblock Here is how I fixed the problem with Rack

Reply:
That's totally fine, we just want to make sure Grape is compatible.